samples/zigbee/light_switch/include/zb_mem_config_custom.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#ifndef ZB_MEM_CONFIG_CUSTOM_H
+#define ZB_MEM_CONFIG_CUSTOM_H 1
+
+
+#define ZB_CONFIG_ROLE_ZED
+/*#define ZB_CONFIG_ROLE_ZR*/
```

Comment:
What about the definitions that are commented out?

Reply:
I'll remove it.